runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java

```diff
@@ -50,7 +50,7 @@
 public class FaloTheBardClue extends ClueScroll implements TextClueScroll, NpcClueScroll
 {
 	private static final List<FaloTheBardClue> CLUES = ImmutableList.of(
-		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", item(DRAGON_SCIMITAR)),
+		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", any("Dragon Scimitar", item(DRAGON_SCIMITAR), item(DRAGON_SCIMITAR_OR))),
```

Comment:
(commenting to commit via GitHub's UI)
```suggestion
		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", any("Dragon scimitar", item(DRAGON_SCIMITAR), item(DRAGON_SCIMITAR_OR))),
```